app/results.py

```diff
@@ -51,6 +51,19 @@ def is_possible_phone_number(query_str):
     return re.search(POSSIBLE_PHONE_NUMBER_RE,
         unicodedata.normalize('NFKC', unicode(query_str)))
 
+def max_word_length(query_words):
+    max_length = max(map(len, query_words))
+    # Count series of CJK characters as one word.
+    cjk_length = 0
+    for i in range(len(query_words)):
+        word = query_words[i]
+        if u'\u3400' <= word <= u'\u9fff':
```

Comment:
Add a comment that this range is copied from the logic in text_query.py. Otherwise it is a bit confusing because this range actually doesn't represent CJK well :)